### ISSUE TITLE
Unable to verify integrity of downloaded node.js binary because no SHA256 checksum was found!

### ISSUE THREAD

**Issue by xiaochao00** [VSCodium/vscodium#2019]
**Describe the bug**
get this error when run build/build.sh

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/docs/index.md) and [the Troubleshooting page](https://github.com/VSCodium/vscodium/blob/master/docs/troubleshooting.md) and my issue is not mentioned there.

**To Reproduce**
1.run script build.sh

**Expected behavior**

**Screenshots**
![image](https://github.com/user-attachments/assets/faa26558-f947-4ad4-854f-673f70a1a971)
**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04

**Additional context**

**Comment by xiaochao00** [comment]
getNodeChecksum fail for node-v20.14.0-glibc-2.17-linux-x64.tar.gz

when I check build/checksums/nodejs.txt here have no: node-v20.14.0-glibc-2.17-linux-x64.tar.gz
1. But code in build/gulpfile.reh.js here checkSum for node-v20.14.0-glibc-2.17-linux-x64.tar.gz
2. How to deal with this,why you add this

**Comment by daiyam** [comment]
You might need `docker` installed. It use it to get the right node version.

**Comment by xiaochao00** [comment]
how can I use docker to fix this? Is there a docker image I can use?

**Comment by xiaochao00** [comment]
@daiyam the day you commit, I find that you fix this in https://github.com/VSCodium/vscodium/pull/2022/files

you remove the VSCODE_NODE_GLIBC in build.sh

**Comment by daiyam** [comment]
> you remove the VSCODE_NODE_GLIBC in build.sh

Yes, `VSCODE_NODE_GLIBC='-glibc-2.17'` is needed for older linux (Ubuntu 16)
